boa_engine/src/context/mod.rs

```diff
@@ -97,6 +97,10 @@ pub struct Context {
     #[cfg(feature = "intl")]
     icu: icu::Icu,
 
+    /// Number of instructions remaining before a forced exit
+    #[cfg(feature = "fuzz")]
+    pub(crate) insns_remaining: usize,
```

Comment:
```suggestion
    pub(crate) instructions_remaining: usize,
```
I would rename this, so its clear what it is :)

Reply:
Completed.